src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -1741,6 +1741,38 @@ public function validateUrl($attribute, $value)
         return preg_match($pattern, $value) > 0;
     }
 
+    /**
+     * Validate that an attribute is a valid URL.
```

Comment:
we are validating a domain, not a URL, so update this comment appropriately.

maybe even add the RFC reference in here.  "according to RFC..."